projects/addon-table/components/table/td/td.component.ts

```diff
@@ -13,9 +8,14 @@ import {NgControl} from '@angular/forms';
     `,
     styleUrls: ['./td.style.less'],
     changeDetection: ChangeDetectionStrategy.OnPush,
+    host: {
+        '[class._editable]': 'current || legacy',
+    },
 })
 export class TuiTdComponent {
-    @HostBinding('class._editable')
-    @ContentChild(NgControl)
-    readonly control: unknown;
+    @ContentChild(TuiControlDirective)
```

Comment:
This is wrong, please remove it completely. That's a different directive, in 3.x we only rely on `AbstractTuiControl` here